wunderlist-create-task.js

```diff
@@ -9,19 +9,49 @@ module.exports = function(RED) {
     this.on('input', function(msg) {
       var wunderlistAPI = wunderlistSDK.getApi(node.config, msg);
       var tasks = wunderlistAPI.http.tasks;
+      var reminders = wunderlistAPI.http.reminders;
 
-      tasks.create({
+      var params = {
         'list_id': Number(n.listId || msg.listId),
         'title': n.taskTitle || msg.payload
-      })
+      };
+      // Add due date if in msg
+      if (msg.due_date){
+        params.due_date = msg.due_date;
+      }
+
+      tasks.create(params)
       .done(function (taskData, statusCode) {
         msg.taskId = taskData.id;
-        node.send(msg);
+        msg.task = taskData;
+
+        // If a reminder is defined
+        if (msg.reminder_date){
```

Comment:
same note about casing of `msg.reminder_date`, please change to `reminderDate`

Reply:
fixed